tests/device_tests/test_msg_change_wipe_code_t1.py

```diff
@@ -174,3 +175,25 @@ def test_set_pin_to_wipe_code(client):
     assert client.features.pin_protection is False
     resp = client.call_raw(messages.GetAddress())
     assert isinstance(resp, messages.Address)
+
+
+def test_set_wipe_code_invalid(client):
```

Comment:
perhaps `@pytest.mark.parametrize("invalid_pin", ("1204", "", "1234567891"))` ? dtto for other test cases

Reply:
Good idea. Done.